pdsfile/pds3file/__init__.py

```diff
@@ -0,0 +1,139 @@
+################################################################################
+# pds3file subpackage & Pds3File subclass with PdsFile as the parent class
+################################################################################
+
+from pdsfile import *
```

Comment:
Need to avoid import *. There should be no need for this,

Reply:
Fixed